client/components/ManageTestQueue/index.jsx

```diff
@@ -89,7 +117,105 @@ const DisclosureContainer = styled.div`
     }
 `;
 
+const CustomToggleButton = styled.button`
+    background-color: transparent;
+    width: 100%;
+    height: 38px;
+    text-align: center;
+
+    border: none;
+    margin: 0;
+    padding: 0;
+    display: block;
+
+    .icon-container {
+        float: right;
+        margin-top: 2px;
+        margin-right: 3px;
+    }
+    .icon-chevron {
+        font-size: 0.8rem;
+    }
+`;
+
+const CustomToggleP = styled.p`
+    border: 1px solid #ced4da;
+    border-radius: 0.375rem;
+    background-color: #fff;
+    padding: 2px;
+    width: 100%;
+    height: 38px;
+    cursor: default;
+    display: inline-block;
+`;
+
+const CustomToggleSpan = styled.span`
```

Comment:
Doesn't seem like the CustomToggle styles are working on Chrome. Tested on Chrome and Firefox.

Reply:
<img width="483" alt="Screenshot 2023-10-16 at 3 40 22 PM" src="https://github.com/w3c/aria-at-app/assets/10382506/f4ad4029-5691-4ea5-b588-baff0e412a5e">

Reply:
Hey @stalgiag the styles will show up in the selection field itself. After clicking on one of the options, the field with the `Phase Selection` placeholder text will show the chosen selection styled like a `<PhasePill />`.